File: garnet/lib/inferior_control/test_main.cc
// Copyright 2019 The Fuchsia Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

// We don't use gtest_main.cc so that we can process logging command line
// arguments.

#include <gtest/gtest.h>
#include <lib/fxl/command_line.h>
#include <lib/fxl/log_settings_command_line.h>

int main(int argc, char **argv) {
  auto command_line = fxl::CommandLineFromArgcArgv(argc, argv);
  if (!fxl::SetLogSettingsFromCommandLine(command_line))
    return 1;

  testing::InitGoogleTest(&argc, argv);
  return RUN_ALL_TESTS();
}

File: examples/dart/fuchsia_modular/slider_mod/lib/test_main.dart
// Copyright 2021 The Fuchsia Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

// TODO(https://fxbug.dev/84961): Fix null safety and remove this language version.
// @dart=2.9

import 'package:flutter_driver/driver_extension.dart';

import 'main.dart' as entrypoint;

void main() {
  enableFlutterDriverExtension();
  entrypoint.main();
}
